### ISSUE TITLE
[Question] Stable version of client

### ISSUE THREAD

**Issue by Bealiah-dev** [elastic/elasticsearch-rs#59]
Hi,

Do you plan to make a stable version of the client ? 
I would prefere use your client in stable version if it possible.
If you don't plan to do it, maybe I'll considere using unstable rust then.

regards,

**Comment by mwilliammyers** [comment]
Hey!

Thanks for your interest in the create!

Yeah we definitely [plan](https://github.com/elastic/elasticsearch-rs/blob/master/CONTRIBUTING.md#current-development-setup) to move to a stable version of the compiler once our dependencies (a git version of `smallvec`* is one of them that I know of) move to stable rust. We also currently use one small nightly feature—`external_doc`, used [here](https://github.com/elastic/elasticsearch-rs/blob/c48b3d828ea9016399d8a8f3ad4ffac813899103/elasticsearch/src/lib.rs#L298)—for including the README code in the doctests. https://github.com/rust-lang/rust/issues/44732 and another issue I have come across but can't find at the moment indicate `external_doc` is fairly close to stabilization...

TL;DR: yes we plan to move to stable but I don't know of an ETA at the moment

*`smallvec` is an indirect dependency of `url` which we depend on:
```
smallvec v1.2.0
└── unicode-normalization v0.1.12
    └── idna v0.2.0
        └── url v2.1.1 (*)
```

**Comment by russcam** [comment]
I'm going to close this issue as @mwilliammyers has answered it perfectly.
